setgrant.js

```diff
@@ -36,29 +41,34 @@ else {
     getHookHashes(account.classicAddress).then(hookHashes => {
```

Comment:
Feels like this will be cleaner if we do this like this
```
getHookHashes(account.classicAddress).then(async hookHashes => {
  let hook2Hashes = await getHookHashes(registryAddress);
  let hook3Hashes = await getHookHashes(heartbeatAddress);
  if (hookHashes && hookHashes.length && hook2Hashes && hook2Hashes.length && hook3Hashes && hook3Hashes.length) {
    ...
  }
}
```